### PR TITLE
Allow boolean dropdowns for contract deploy

### PR DESCRIPTION
**Pull request by jacogr**
- Allow boolean dropdown for contract deploys, Fixes https://github.com/ethcore/parity/issues/3067
- Make uint non-hex to align with execute modal, Closes https://github.com/ethcore/parity/issues/3068

### CODE REVIEW

**Comment by chevdor** [comment]
tested, confirmed fixed. Thanks @jacogr

File: js/src/modals/DeployContract/DetailsStep/detailsStep.js [hunk 1]
```diff
@@ -115,6 +117,24 @@ export default class DetailsStep extends Component {
...
+                value={ bool }
+                label={ bool }>{ bool }</MenuItem>
+            );
+          });
+          inputBox = (
+            <Select
+              label={ label }
+              value={ params[index] ? 'true' : 'false' }
+              error={ paramsError[index] }
+              onChange={ onChangeBool }>{ boolitems }</Select>
```
Comment (derhuerst) [inline js/src/modals/DeployContract/DetailsStep/detailsStep.js]: multiple lines for better readability? (:

File: js/src/modals/DeployContract/DetailsStep/detailsStep.js [hunk 2]
```diff
@@ -168,8 +188,12 @@ export default class DetailsStep extends Component {
             params.push('');
             break;
 
+          case 'bool':
```
Comment (derhuerst) [inline js/src/modals/DeployContract/DetailsStep/detailsStep.js]: `inputs.forEach` -> `inputs.map`

Comment (jacogr) [inline js/src/modals/DeployContract/DetailsStep/detailsStep.js]: It is parameters according the the API, so it is consistent all the way through.

**Pull request merged**